docs/modules/concepts/pages/storage/users.adoc

```diff
@@ -0,0 +1,59 @@
+= User Directory
+:navtitle: Users
+
+== Overview
+
+Whereas xref:storage/mailbox.adoc[mailboxes] are clearly a core component of
+any email system, user management is a lighter shade of gray.
+
+On the one hand, mailboxes ought to match up with actual users, so there is
+a clear relationship between mailbox management and user management.
+Because of this relationship, it is not at all strange to include a user directory
+directly within an email system.
+
+However, on the other hand email is only one of many applications that require
+user management. It is therefore also completely reasonable to use some kind of
+centralized user directory, and ensure that mailboxes somehow synchronize with
+this external system.
+
+The former case is much simpler to manage, but requires duplicate and therefore
+unproductive work. The latter case can become quite complex from a technological 
+perspective (and all the risks that are associated with it), but reduces the
+amount of unproductive work.
+
+== Common Operations
+
+Because the user directory is used in the context of managing mailboxes, there
+are some operations that are relatively common to any email system.
+
+=== Domain Management
+
+A mailbox belongs to a https://en.wikipedia.org/wiki/Domain_name[domain]. For a mail
+system that can process multiple domains, the system will usually have operations to:
+
+* List available domains
+* Add a domain
+* Remove a domain
+
+
+=== User Management
+
+A User has at least one mail box, and will usually have a canonical mailbox
+that matches the user's identity in the system.
+For each domain that is managed by the system, it is common to be able to:
+
+* List all users for the domain
+* Add a user
+* Remove user 
+
+
+=== Other Common Operations
+
+It is common to allow user's to have one or more
+https://en.wikipedia.org/wiki/Email_alias[aliases].
+
+Some systems provide an https://en.wikipedia.org/wiki/Autoresponder[autoresponder]
+(or "vacation") facility to its users.
```

Comment:
This "auto responder" is either implemented via SIEVE scripts, or via JMAP. Maybe it should be coming after SIEVE scripts.

Reply:
Ok. I was kinda grasping at straws by this point anyway. :)

Reply:
I updated the text. Hopefully this works.